src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -354,6 +360,12 @@ class GcsPlacementGroupManager : public rpc::PlacementGroupInfoHandler {
   // Update placement group load information so that the autoscaler can use it.
   void UpdatePlacementGroupLoad();
 
+  /// Check if this placement group is waiting for scheduling.
+  bool IsPlacmentGroupIDInPendingQueue(const PlacementGroupID &placement_group_id) const;
```

Comment:
no strong opinion, but feel the name is too long.
```
IsInPendingQueue(...)
```

Reply:
done